compact/display.xml

```diff
@@ -49,15 +51,15 @@
     <vis name="TOFCHoneycombVis"  ref="TrackerServiceVis"   visible="true"  showDaughters="true"/>
 
 
-    <vis name="FFTrackerVis"        ref="AnlRed" />
-    <vis name="FFTrackerSupportVis" ref="AnlBlue"             visible="true"  showDaughters="false" />
-    <vis name="FFTrackerShieldingVis" ref="AnlGray"           visible="true"  showDaughters="false" />
-    <vis name="FFTrackerServiceVis" ref="AnlGold"             visible="true"  showDaughters="false" />
-    <vis name="FFTrackerLayerVis"   ref="TrackerVis"          visible="true"  showDaughters="true" />
-    <vis name="FFTrackerModuleVis"  ref="FFTrackerLayerVis"   visible="true"  showDaughters="true" />
-    <vis name="FFTrackerSurfaceVis" ref="FFTrackerLayerVis"   visible="true"  showDaughters="true" />
+    <vis name="FFTrackerVis"        alpha="1"  ref="AnlRed" />
+    <vis name="FFTrackerSupportVis" alpha="1"  ref="AnlBlue"             visible="true"  showDaughters="false" />
+    <vis name="FFTrackerShieldingVis" alpha="1" ref="AnlGray"             visible="true"  showDaughters="false" />
+    <vis name="FFTrackerServiceVis" alpha="1"  ref="AnlGold"             visible="true"  showDaughters="false" />
+    <vis name="FFTrackerLayerVis"  alpha="1"   ref="TrackerVis"          visible="true"  showDaughters="true"  />
+    <vis name="FFTrackerModuleVis" alpha="1"   ref="FFTrackerLayerVis"   visible="true"  showDaughters="true"  />
+    <vis name="FFTrackerSurfaceVis" alpha="1"  ref="FFTrackerLayerVis"   visible="true"  showDaughters="true"  />
     <comment> For shielded modules by default just display to module instead of 6 layers </comment>
-    <vis name="FFTrackerShieldedModuleVis" ref="FFTrackerModuleVis"   visible="true"  showDaughters="false" />
+    <vis name="FFTrackerShieldedModuleVis" alpha="1" ref="FFTrackerModuleVis"   visible="true"  showDaughters="false" />
```

Comment:
Please put visualization changes into separate PR. Applies to many areas of this file.